modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/util/UploadImageUtil.java

```diff
@@ -34,7 +34,11 @@ public static long getMaxFileSize(PortletRequest portletRequest) {
 		ThemeDisplay themeDisplay = (ThemeDisplay)portletRequest.getAttribute(
 			WebKeys.THEME_DISPLAY);
 
-		if (StringUtil.startsWith(
+		String type = portletRequest.getParameter("type");
+
+		if (StringUtil.equals(type, "organization_portrait") ||
+			StringUtil.equals(type, "user_portrait") ||
```

Comment:
and put these conditions after the other (so they have the same sort of the variable declaration)